src/www/ui/page/AdminLicenseCandidate.php

```diff
@@ -61,6 +67,14 @@ function __construct()
    */
   protected function handle(Request $request)
   {
+    $action = $request->get('action');
+    $deleteRfPk = intval($request->get('deleteRfPk'));
```

Comment:
Same as above, maybe use the already used filed `'rf'` instead of the new `'deleteRfPk'`